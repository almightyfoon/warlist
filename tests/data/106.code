ca501b_-0tcVaPaPaPaQbvbfaObbbebeb9b9___-0uaCg_aReKbbfUfUcoaYa_b0