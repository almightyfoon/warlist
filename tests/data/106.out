https://conflictchamber.com/#ca501b_-0tcVaPaPaPkHbvbfaObbbebeb9b9___-0uaCg_aReKbbfUfUcoaYa_b0

Legion Army - 78 / 75 points
[Theme] Oracles of Annihilation

!!! This army must contain 75 points or fewer

[Lylyth 3] Lylyth, Reckoning of Everblight [+29]
 - Angelius [17]
 - Angelius [17]
 - Angelius [17]
 - Carnivean [18]
 - Naga Nightlurker [8]
 - Nephilim Bolt Thrower [11]
 - Seraph [14]
Blighted Nyss Shepherd [1]
Blighted Nyss Sorceress & Hellion [0(6)]
Blighted Nyss Sorceress & Hellion [0(6)]
The Forsaken [0(4)]
The Forsaken [4]

Legion Army - 71 / 75 points
[Theme] Children of the Dragon

[Thagrosh 1] Thagrosh, Prophet of Everblight [+28]
 - Azrael [21]
 - Typhon [20]
 - Zuriel [18]
Blighted Nyss Shepherd [1]
Blighted Nyss Warlord [0(5)]
Blighted Nyss Warlord [5]
Spell Martyr [1]
Blighted Nyss Raptors (max) [18]
Blighted Nyss Swordsmen (max) [15]
 - Blighted Nyss Swordsman Abbot & Champion [0(6)]
